src/clientdlg.cpp

```diff
@@ -748,10 +734,10 @@ void CClientDlg::OnConnectDlgAccepted()
 void CClientDlg::OnConnectDisconBut()
 {
     // the connect/disconnect button implements a toggle functionality
-    if ( pClient->IsRunning() )
+    if ( pClient->Disconnect() )
```

Comment:
To be refactored to not have an empty if() -- just style.

Reply:
This should just call `pClient->Disconnect()`, I think.  When that emits "Disconnected", the client dialog can clean up the mixer board.